integration/docker/run_test.go

```diff
@@ -204,3 +204,34 @@ var _ = Describe("run", func() {
 		})
 	})
 })
+
+var _ = Describe("run", func() {
+	var (
+		args   []string
+		id     string
+		period uint
+		quota  uint
+		vCPUs  uint
+	)
+
+	BeforeEach(func() {
+		quota = 50000
+		period = 20000
+		vCPUs = (quota + period - 1) / period
+		id = RandID(30)
+		args = []string{"--rm", "--name", id, "--cpu-quota", fmt.Sprintf("%d", quota),
```

Comment:
Just to note, according to the Docker docs, it seems `quota` and `period` may be semi-deprecated
https://docs.docker.com/engine/admin/resource_constraints/#cpu
https://blog.docker.com/2017/01/cpu-management-docker-1-13/

Maybe we should re-write, or at least add, `--cpus` based tests (which I believe will be translated by docker into `quota` and `period` values to the runtime itself?)

Reply:
ohh I see, thanks @grahamwhaley